components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.cache.ui/src/main/resources/org/wso2/carbon/mediator/cache/ui/i18n/Resources.properties

```diff
@@ -14,4 +14,6 @@ mediator.cache.protocoldetails=Protocol Details
 mediator.cache.protocol=Protocol Type
 mediator.cache.httpmethods=HTTP Methods To Cache (comma separated list)
 mediator.cache.headersToExclude=Headers To Exclude In Hash (comma separated list)
-mediator.cache.responsecodes=Response Codes (regex expression)
\ No newline at end of file
+mediator.cache.responsecodes=Response Codes (regex expression)

```

Comment:
Formatting issue shall we fix this?

Reply:
Normally, we don't place space with '='. I followed the same as in the other products. Do you think we need to change this?